src/identity-providers/ManageOrderDialog.tsx

```diff
@@ -101,48 +137,46 @@ export const ManageOrderDialog = ({
         <Text>{t("orderDialogIntro")}</Text>
       </TextContent>
 
-      <DataList
-        aria-label={t("manageOrderTableAria")}
-        data-testid="manageOrderDataList"
-        isCompact
-        onDragFinish={onDragFinish}
-        onDragStart={onDragStart}
-        onDragMove={onDragMove}
-        onDragCancel={onDragCancel}
-        itemOrder={order}
-      >
-        {sortBy(providers, "config.guiOrder").map((provider) => (
-          <DataListItem
-            aria-labelledby={provider.alias}
-            id={`${provider.alias}-item`}
-            key={provider.alias}
+      <DragDrop onDrag={onDrag} onDragMove={onDragMove} onDrop={onDrop}>
+        <Droppable hasNoWrapper>
+          <DataList
+            aria-label={t("manageOrderTableAria")}
+            data-testid="manageOrderDataList"
+            isCompact
           >
-            <DataListItemRow>
-              <DataListControl>
-                <DataListDragButton
-                  aria-label="Reorder"
-                  aria-labelledby={provider.alias}
-                  aria-describedby={t("manageOrderItemAria")}
-                  aria-pressed="false"
-                />
-              </DataListControl>
-              <DataListItemCells
-                dataListCells={[
-                  <DataListCell
-                    key={`${provider.alias}-cell`}
-                    data-testid={provider.alias}
-                  >
-                    <span id={provider.alias}>{provider.alias}</span>
-                  </DataListCell>,
-                ]}
-              />
-            </DataListItemRow>
-          </DataListItem>
-        ))}
-      </DataList>
-      <div className="pf-screen-reader" aria-live="assertive">
-        {liveText}
-      </div>
+            {order.map((alias) => (
+              <Draggable key={alias} hasNoWrapper>
+                <DataListItem
+                  aria-labelledby={alias}
+                  id={`${alias}-item`}
+                  ref={React.createRef()}
+                >
+                  <DataListItemRow>
+                    <DataListControl>
+                      <DataListDragButton
+                        aria-label="Reorder"
+                        aria-labelledby={alias}
+                        aria-describedby={t("manageOrderItemAria")}
+                        aria-pressed="false"
+                      />
+                    </DataListControl>
+                    <DataListItemCells
+                      dataListCells={[
+                        <DataListCell key={`${alias}-cell`} data-testid={alias}>
```

Comment:
Since keys do not need to be globally unique I think you can get away with passing this value directly.
```suggestion
                        <DataListCell key={alias} data-testid={alias}>
```